src/main/resources/bootstrap.yml

```diff
@@ -23,3 +27,20 @@ spring:
     enabled: true
     sampler:
       probability: 1.0
+---
+spring:
+  profiles: with_consul
+  cloud:
+    consul:
```

Comment:
we should disable spring.cloud.consul.config.enabled=false

Reply:
config is not in gradle dependencies

Reply:
but just to be sure I will add this